1346_check_if_its_double_exist.py

```diff
@@ -0,0 +1,17 @@
+from typing import List
+
+
+class Solution:
+    def checkIfExist(self, arr: List[int]) -> bool:
+        
+        s = set(arr)
```

Comment:
Name the variable a descriptive name, num_set for example.
It's also the same as for SS declared below.